api/clients/disperser_client.go

```diff
@@ -10,6 +10,7 @@ import (
 
```

Comment:
This client is only compatible with v1 disperser. Do we need this change if we don't support payments in v1?

Reply:
I think my impression for what we want to do next is finishing the code but will not enable the feature flag?

Reply:
Not sure if we need this complexity if we're not going to turn it on v1. 
Especially it changes API for any external package using this client. When we remove `accountant` from this client, they would have to update how they use this client

Reply:
I don't think we would need to remove accountant from v1 client, and they would need to update for v2 anyways, and we could simply set accountant to always be nil? any way, I removed it for now and can implement for v2 later on.
I also removed the API from disperser server. if it makes the review process cleaner, I could make a separate PR that focus on removing the payment APIs